metadata.json

```diff
@@ -46,7 +46,8 @@
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-
+        "10",
```

Comment:
was your intention to add debian 10 and 11, or did you mean 11 and 12?

Reply:
Should be 11 and 12 at this point. Updated it.